admin/assets/javascripts/global-scripts.js

```diff
@@ -642,4 +642,28 @@ $(function () {
       $('#' + $(this).context.id).text('(' + totalPupils + ' pupil' + (totalPupils > 1 ? 's' : '') + ')')
     })
   }
+
+  /**
+   * Filtering pupils by name.
+   */
+  if ($('.filter-name').length > 0) {
```

Comment:
@danwilliams @jon-shipley @GuyHarwood I am worried global scripts is becoming quite large and not easily maintainable. Would it be a right time to start moving to separate files depending on purpose?

Reply:
@GeorgeChatzigiannis yes, absolutely